infrastructure/cover-service-cron-jobs/templates/cron-jobs.yaml

```diff
@@ -88,6 +88,21 @@ spec:
                     secretKeyRef:
                       name: {{ .Release.Name }}-secret
                       key: APP_SECRET
+                - name: APP_FORSIDERDK_USER
+                  valueFrom:
+                    secretKeyRef:
+                      name: {{ .Release.Name }}-secret
+                      key: APP_FORSIDERDK_USER
+                - name: APP_FORSIDERDK_PASSWORD
+                  valueFrom:
+                    secretKeyRef:
+                      name: {{ .Release.Name }}-secret
+                      key: APP_FORSIDERDK_PASSWORD
+                - name: APP_FORSIDERDK_SUBFOLDERS
```

Comment:
This should no be here... if it's not a secrect. The whole configmap is loaded in another way

Reply:
Removed